packages/markdown/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -56,8 +56,30 @@ exports[`should strip out inputs 1`] = `""`;
 
 exports[`tables 1`] = `
 "<div class=\\"marked-table\\"><table>
-<thead><tr><th>Tables</th><th style=\\"text-align:center\\">Are</th><th style=\\"text-align:right\\">Cool</th></tr></thead>
-<tbody><tr><td>col 3 is</td><td style=\\"text-align:center\\">right-aligned</td><td style=\\"text-align:right\\">$1600</td></tr><tr><td>col 2 is</td><td style=\\"text-align:center\\">centered</td><td style=\\"text-align:right\\">$12</td></tr><tr><td>zebra stripes</td><td style=\\"text-align:center\\">are neat</td><td style=\\"text-align:right\\">$1</td></tr></tbody>
+<thead>
+<tr>
+<th>Tables</th>
+<th align=\\"center\\">Are</th>
```

Comment:
What is it that adds these attributes instead of `text-align: center`? These are deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-align